### ISSUE TITLE
greedy_iou tp_list and fp_list has some predictions in both lists

### ISSUE THREAD

**Issue by Matze99** [DigitalSlideArchive/HistomicsDetect#50]
code for replication

'
ious, _ = iou(boxes, rpn_boxes)

precision, recall, tp, fp, fn, tp_list, fp_list, fn_list = greedy_iou(ious, 0.18)

indexes = tf.expand_dims(tp_list[:, 1], axis=1)
labels = tf.scatter_nd(indexes, tf.ones(tf.shape(indexes)), [tf.shape(rpn_boxes)[0], 1])

actual_fp = tf.reduce_sum(tf.cast(labels == 0, tf.int32))
'
